.github/workflows/claude.yml

```
@@ -30,6 +30,16 @@ jobs:
         with:
           fetch-depth: 1
 
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version-file: go.mod
+
+      - name: Install golangci-lint
+        uses: golangci/golangci-lint-action@v7
+        with:
+          install-only: true
```

actionlint (1.7.11)

[error] input "install-only" is not defined in action "golangci/golangci-lint-action@v7". available inputs are "args", "cache-invalidation-interval", "github-token", "install-mode", "only-new-issues", "problem-matchers", "skip-cache", "skip-save-cache", "verify", "version", "working-directory"

(action)